CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ target_link_libraries(project1 project1_lib)
 ################################
 if (test)
   # This adds another subdirectory, which has 'project(gtest)'.
+  set( gtest_force_shared_crt ON CACHE BOOL "Always use msvcrt.dll" )
```

Comment:
Nm, looks like this is fixed in https://github.com/dmonopoly/gtest-cmake-example/pull/6/files/bf56234fcb653ad4f9b9420df7946eb79da7d1d0